README.md

```diff
@@ -37,6 +37,9 @@ are `systemd-devel`, `json-c-devel`, and `libcurl-devel` on RPM-based distros,
 and `libjson-c-dev`, `libsystemd-journal-dev`/`libsystemd-dev` and
 `libcurl-*-dev` on Debian-based distros.
 
+To build from Git on RPM-based distros you'll need those packages: `autoreconf`, `automake`, `libtool`.
```

Comment:
I think it should be "these", not "those", but perhaps even better could be just "you'll need `autoreconf`, `automake`, and `libtool` packages." Please also wrap the line on 78 columns.